.travis.yml

```diff
@@ -19,12 +19,12 @@ install:
   case $TRAVIS_OS_NAME in
   linux)
     curl https://dist.crystal-lang.org/apt/setup.sh | sudo bash
-    sudo apt-get install -y crystal libgmp-dev llvm-3.8-dev build-essential
+    sudo apt-get install -y --allow-unauthenticated crystal libgmp-dev llvm-3.8 llvm-3.8-dev build-essential
```

Comment:
Installing [`llvm-3.9-dev`](https://packages.ubuntu.com/trusty/llvm-3.9-dev), instead of `llvm-3.8` and `llvm-3.8-dev`, should be sufficient.

Reply:
But, I already tried this :point_right: https://github.com/crystal-lang-tools/scry/pull/24/commits/0e6b83b110ffe93c9df6ea48e42a17008e94c58d